pkg/cloud/csvprovider.go

```diff
@@ -33,6 +33,8 @@ type CSVProvider struct {
 	NodeMapField            string
 	PricingPV               map[string]*price
 	PVMapField              string
+	GPUClassPricing         map[string]*price
+	GPUMapFields            []string
```

Comment:
What is `GPUMapFields`?

Reply:
This is just for my own understanding!

Reply:
This represents the possible fields in the node.labels that could correspond to a GPU. I can drop a comment.